client/src/view/island-component.tsx

```diff
@@ -252,6 +257,17 @@ export class IslandComponent extends React.Component<IslandComponentProps, objec
         return geometry
     }
 
+    public get actionHexalotId(): string | undefined {
```

Comment:
What's this for? Why the name?

Reply:
Weird name, true.  Will change it to `activeHexalotId`.  The seeds are rendered, except for the one that has been launched for driving or evolution.